package.json

```diff
@@ -9,7 +9,7 @@
     "build": "npm run clean && tsc --project tsconfig.release.json",
     "build:test": "npm run clean && tsc --project . && mkdirp build/test && ncp test/test-files build/test/test-files",
     "lint": "resin-lint --typescript src/ test/ typings/ && tsc --noEmit",
-    "prettify": "prettier --config ./node_modules/resin-lint/config/.prettierrc --write \"{src,test,typings}/**/*.ts\"",
+    "prettify": "resin-lint --typescript --fix src/ test/ typings/",
```

Comment:
* I am not keen on this change to `prettify` because `resin-lint` does more than prettifying, and hides prettier's output (which would show which files were prettified).
* Shouldn't `--fix` be added to the lint script?
* Neither `lint` nor `prettify` seems to be called by any other npm scripts. Perhaps `npm run lint` (including `--fix`) could be part of the `npm run build` script.

Reply:
* `resin-lint --fix` does do more, but it's all things that are required to be done regardless, the reason we only used `prettier` before was because it automated as much as we could at the time and nowadays we have resin-lint to handle that and more :)
* `--fix` is only in the `prettify` script to have the same semantics between the read-only and read-write versions, it's also necessary for CI tests to do the read-only check
* Yeah it should definitely be run as part of the CI, I've added the lint check to the test command so the CI will enforce it now